google_api_availability_android/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
  * Updates gradle version from 8.1.0 to 8.10.2
  * Updates android.application version from 8.1.0 to 8.7.0
+ * Fixed publishing warning'>=2.15.0 <3.0.0' to '>=2.15.0 <4.0.0'.
```

Comment:
The version number changed in the last commit, but forgot to update the message in the changelog:

```suggestion
* Fixed publishing warning and updated Dart SDK dependency from '>=2.15.0 <3.0.0' to '^3.5.0'.
```